src/test/java/seedu/address/model/person/DescriptionTest.java

```diff
@@ -0,0 +1,39 @@
+package seedu.address.model.person;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+public class DescriptionTest {
+
+
+    @Test
+    public void constructor_invalidDescription_throwsIllegalArgumentException() {
+        String invalidDescription = "";
+        assertThrows(IllegalArgumentException.class, () -> new Description(invalidDescription));
+    }
+
+    @Test
+    public void constructor_invalidDescription_emptySpaces_throwsIllegalArgumentException() {
```

Comment:
Not sure if I'm missing something but I thought test methods had a 3-part format: Underscores may be used in test method names using the following three part format featureUnderTest_testScenario_expectedBehavior()

Reply:
Right, thanks for spotting the error! Updated accordingly